.gitignore

```diff
@@ -1,10 +1,21 @@
-tags
+# Ignore everything
```

Comment:
it is preferable to add files or folders to be ignored to the .gitignore instead of a list of files to not ignore...

Reply:
I think it would be better this way b/c the "$gradle bootRun" creates so much junk, and the potentially new things we add will create more junk. So saying "ignore everything but these 3 files" would require less maintenance, instead of editing the .gitignore whenever we make certain additions. It is suppose to reduce the amount of useless files in our github, however it seems due to our start we already have some (not like it matters, but ideally it wouldnt be there)

Reply:
I'll change it for the commit and so we could get things done though. We could continue the super important discussion of how a .gitignore should be structured some other time.